tron/bin/check_tron_jobs.py

```diff
@@ -251,7 +251,7 @@ def is_job_stuck(
 ):
     next_run_time = None
     for job_run in job_runs:
-        states_to_check = {"running", "waiting"}
+        states_to_check = {"running", "waiting", "starting"}
```

Comment:
heh, i like how we're pretty inconsistent re: sets and lists in this file :p

Reply:
we could also move this outside the loop so that we're not re-creating a set multiple times, but it's probably fine to punt on refactoring this script :p

Reply:
Hm good call, though i also briefly saw we had https://github.com/Yelp/Tron/blob/master/tron/core/jobrun.py#L456 which makes me wonder if we should just reuse this and why didn't we already, but agree I'll leave the archaeology for another time.